projects/packages/forms/src/dashboard/components/layout/index.js

```diff
@@ -1,18 +1,58 @@
 import { JetpackFooter } from '@automattic/jetpack-components';
 import { shouldUseInternalLinks } from '@automattic/jetpack-shared-extension-utils';
+import { TabPanel } from '@wordpress/components';
+import { useCallback } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import clsx from 'clsx';
+import { Outlet, useLocation, useNavigate } from 'react-router-dom';
+import { config } from '../../index';
 import JetpackFormsLogo from '../logo';
 
 import './style.scss';
 
-const Layout = ( { children, className, showFooter } ) => {
+const Layout = ( { className, showFooter } ) => {
+	const location = useLocation();
+	const navigate = useNavigate();
+
+	const getCurrentTab = () => {
+		const path = location.pathname.split( '/' )[ 1 ];
```

Comment:
Should we be able to pass in any sort of path? Or should we limit it a subset of possible paths?